str/associatr/pseudobulk.py

```diff
@@ -25,13 +26,19 @@
 from cpg_utils.hail_batch import get_batch, image_path, output_path
 
 
-def pseudobulk(input_file_path, target_sum, min_pct):
+def pseudobulk(input_file_path, id_file_path, target_sum, min_pct):
     """
     Performs pseudobulking in a cell-type chromosome-specific manner
     """
     expression_h5ad_path = to_path(input_file_path).copy('here.h5ad')
     adata = sc.read_h5ad(expression_h5ad_path)
 
+    # retain only samples in the id file
+    with to_path(id_file_path).open() as csvfile:
```

Comment:
how many IDs are in each cohort? it might be easier to add a list of SG IDs into a config file, and access them that way instead of having to copy a file into GCP then parse it etc.

Reply:
It's >900 each time. I prefer to keep it separate as a CSV file so the TOML is cleaner